src/patternfly/demos/PageLayout/examples/page-layout-default-nav-example.hbs

```diff
@@ -1,20 +1,15 @@
 {{#> background-image}}
 {{/background-image}}
-
 {{#> page page--attribute='id="page"'}}
   {{#> page-header}}
     {{!-- Brand --}}
     {{#> page-header-brand}}
-      {{#> page-sidebar-toggle button-toggle--interactive="true" nav-toggle--attribute='id="nav-toggle" aria-label="Toggle primary navigation"'}}
-      {{/page-sidebar-toggle}}
-      <a class="pf-l-page__header-brand-link" href="#">
-        {{#> brand brand--src="/assets/images/l_pf-reverse.svg" brand--alt="Patternfly Logo" brand--modifier="pf-m-patternfly-logo"}}{{/brand}}
-      </a>
+      {{#> page-template-header-brand-elements page-template-header-brand--toggle="true"}}
+      {{/page-template-header-brand-elements}}
     {{/page-header-brand}}
-    {{#> page-template-header-elements}}
-    {{/page-template-header-elements}}
+    {{#> page-template-header-tools-elements}}
+    {{/page-template-header-tools-elements}}
   {{/page-header}}
-
   {{!-- Nav --}}
   {{#> page-sidebar}}  
     {{#> nav nav--vertical="true" nav--attribute='id="primary-nav" aria-label="Primary Nav Default Example"'}}
```

Comment:
nav--vertical="true" is no longer needed

Reply:
removed